src/handlers/events/delivery/build/FindArtifactOnImageLinked.ts

```diff
@@ -16,11 +16,15 @@
 
 import { EventFired, EventHandler, HandleEvent, HandlerContext, HandlerResult, logger, Success } from "@atomist/automation-client";
 import { subscription } from "@atomist/automation-client/graph/graphQL";
+import {
+    ArtifactStore,
+    ProjectLoader,
+    RepoRefResolver,
+} from "../../../..";
```

Comment:
You need to delete the index file to get rid of this imports from it. Then rerun `npm:generate-barrels`